### ISSUE TITLE
Understanding Training metrics

### ISSUE THREAD

**Issue by bibekyess** [KaihuaTang/Scene-Graph-Benchmark.pytorch#182]
## ❓ Questions and Help
Hello!
I have some general questions on understanding the metrics used in training. From the example below:

`2022-08-13 19:12:26,520 maskrcnn_benchmark INFO: eta: 1:05:16  iter: 26000  loss: 0.0018 (0.0215)  loss_refine_att: 0.0003 (0.0013)  loss_refine_obj: 0.0014 (0.0201)  loss_rel: 0.0000 (0.0002)  time: 0.9628 (0.9792)  data: 0.0123 (0.0197)  lr: 0.000009  max mem: 8478
`
What is this general `loss`? And how is it different from the loss value inside parenthesis `()` and what about `data`? What does `data` here refer to?
I would greatly appreciate your answer.

Thank you and have a good day!
